scrapers_next/pa/committees.py

```diff
@@ -0,0 +1,171 @@
+from spatula import CSS, HtmlPage, HtmlListPage, XPath, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class HouseCommitteeDetail(HtmlPage):
+    example_source = "https://www.legis.state.pa.us/cfdocs/CteeInfo/index.cfm?Code=32&CteeBody=H&SessYear=2021"
+    example_name = "Aging & Older Adult Services"
+    example_input = ScrapeCommittee(
+        name=example_name, classification="committee", chamber="lower"
+    )
+
+    def process_page(self):
+        com = self.input
+        try:
+            # This section has the chair memebers the regular, democratic and minority and the roles
+            # main chair
+            Chair_Member = (
+                CSS("div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText a")
+                .match(self.root)[0]
+                .text.strip()
+            )
+            # main chair role
+            Chair_Member_rolez = (
```

Comment:
We talked about variable naming conventions last week, and you have only implemented a couple of the pieces of feedback I gave you about variables. Two pieces that are missing:

- we talked about `rolez` and how that is not an English word. If you mean `roles` it needs to be `roles`
- Python style guide says that variable names in functions use lowercase letters. So this variable needs to be `chair_member_roles` (and other variables throughout need to follow this style). I pointed out the [PEP 8 python style guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names): that needs to be followed throughout, please make changes.